### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by vr8hub** [BR903/israel-zangwill_the-big-bow-mystery#1]
- [ ] Typogrify makes a change that should be accepted.
- [ ] Semanticate makes a change to chapter 10 that should be accepted.
- [ ] As lint correctly reports, the h2 in the Introduction requires semantics. The h2 would be the title, the h3 the subtitle.
- [ ] The Introduction would appear to be non-fiction? As would the Note?
- [ ] The letter in the introduction should have those semantics throughout, with the additional standard CSS added as well. Also note the two closing `</i>`'s.
- [ ] Also note that CSS is needed so that the newspaper name is roman text (italics within italics are generally roman).
- [ ] It's not strictly necessary, so up to you, but given the spacing in the scans for "Yours, etc.", we would typically put that on its own line. Either way, it should have z3998:valediction markup.
- [ ] The filename and id of the preface should be preface; see [SEMoS 2.2.4.5](https://standardebooks.org/manual/1.7.1/single-page#2.2.4.5).
- [ ] Generally speaking, we prefer formatting with CSS rather than markup when an entire element is being formatted. Since there are only three blockquotes in chapters 6/11, and they're the only blockquotes in chapters 6/11, they're pretty simple to target without the class. Also, since CSS is being created for them anyway, then it should handle the small-caps as well, without the `<b>` markup. (The latter would be true even if a class was being used.)
- [ ] This would apply to the blockquote headers in chapter 4 as well; they can be targeted and formatted in CSS without needing the markup.
- [ ] We prefer to keep commits focused on a single task, so the Koh-i-noor fix in chapter 8 would ideally be in its own commit, since it has nothing to do with the rest of that commit. It's not a big deal, though, so no change needed, just please make a note for the future.
- [ ] The letter in chapter 10 should have letter semantics throughout, i.e. the letter itself and all of its parts.
- [ ] ipsissima verba and rencontre are in M-W and thus do not need tags. Ex machina is just shorthand for deus ex machina, which is also in M-W.
- [ ] Spelling throughout appears to be mostly US (color, favor*, center, etc.), so xml:lang should be en-US.
- [ ] According to Kleineh's German Wiki entry, his fullname was Oscar Konrad Kleineh, so he should have a full-name entry in the metadata.
- [ ] There are a few small numbers (10 guineas, 18 feet, etc.) that should be spelled out, editorial. See [SEMoS 8.8.3](https://standardebooks.org/manual/1.7.1/single-page#8.8.3).
- [ ] In chapter 4, the first blockquote heading has no opening quote, while the remaining two do. That is the way they are in the scans as well, but that seems (is) inconsistent. The first one should probably also have an opening quote (editorial).

**Comment by BR903** [comment]
> As lint correctly reports

I had (and still have) a clean bill of health from lint. It's version 2.6.0; is there a later version?

**Comment by BR903** [comment]
Ready for re-review.

The "Koh-i-noor" change was a pure typo that I commited accidentally and didn't notice until later. I attempted to undo the change with a rebase, but it looks like targeted the wrong commit. I'll accept your offer to leave it be, rather than risk confusing things further.

Also, I took the liberty of including two other changes with this set of commits, both in content.opf. I added the Haycraft Queen Cornerstones metadata, naturally, but I also cleaned up some infelicities in my original long description. Apologies for the last-minute twiddling.

**Comment by vr8hub** [comment]
> I had (and still have) a clean bill of health from lint. It's version 2.6.0; is there a later version?

Oops, so sorry, I was accidentally using an experimental one that has the changes for hgroup in it. I apologize for the confusion.

It was right, however. :)

**Comment by vr8hub** [comment]
> Also, I took the liberty of including two other changes with this set of commits, both in content.opf. I added the Haycraft Queen Cornerstones metadata, naturally, but I also cleaned up some infelicities in my original long description. Apologies for the last-minute twiddling.

No worries. I’ll relook at everything later today.

**Comment by vr8hub** [comment]
Two very small things and it's done.

I happened to notice this time that part of the postscript of the second letter in chapter 4 is a quote from another letter. As such those two paragraphs should be in a letter blockquote as well. (Editorial)

Although the `header` CSS works as is, I think it would be clearer if it were at least `blockquote header`, if nothing else to communicate what headers are being targeted. And maybe not in the letter section of the CSS, so it's clear it's not letters being targeted?

**Comment by vr8hub** [comment]
Oh, sorry, I forgot—the item on the small numbers was checked off, but they are still present.

**Comment by BR903** [comment]
> Oh, sorry, I forgot—the item on the small numbers was checked off, but they are still present.

I don't know how that happened, but clearly I should be fired.

> I happened to notice this time that part of the postscript of the second letter in chapter 4 is a quote from another letter. As such those two paragraphs should be in a letter blockquote as well. (Editorial)

The double-indentation leaves a very narrow text column on my readers. I don't suppose it would be acceptable to have the inner blockquote without extra margins?

**Comment by vr8hub** [comment]
> The double-indentation leaves a very narrow text column on my readers. I don't suppose it would be acceptable to have the inner blockquote without extra margins?

Yeah, I thought about that; I checked it on my iPhone, and although it wasn't great, it wasn't horrible, at least with the font size I use. But, I'm not opposed; just set `blockquote blockquote` left/right margin to 0 (we do want the vertical break), and we'll see if if it passes muster with Alex. It doesn't cost anything to try. :)

**Comment by BR903** [comment]
Thanks. Okay, I think that's everything.
